bin/deploy

```diff
@@ -11,19 +11,7 @@ set -e
 
 case "${1:-carnival-staging}" in
   carnival-staging)
-    git push --force staging master
-
-    if [ -n "$BUILD" ]; then
-      heroku run --size=PX build
-
-      git checkout -b deploy
-      git commit --allow-empty -m "Trigger deployment"
-      git push --force staging deploy:master
-      git checkout master
-      git branch -D deploy
-
-      heroku ps:scale web=1
-    fi
+    heroku docker:release
```

Comment:
@jferris this line is probably missing an `--app carnival-staging`.

@tonyd256 Among other things https://github.com/heroku/heroku-docker/blob/master/commands/release.js#L43

IMO the fact that it requires a compose file and runs `docker-compose build web` instead of just `docker build` is bizarre. It also requires an `app.json` so it can sort out add-ons instead of only doing that *if* there's an `app.json`. A little annoying that you need the extra boilerplate, but oh well.